js/iframe-coordinator/package.json

```diff
@@ -13,13 +13,15 @@
   ],
   "devDependencies": {
     "elm-webpack-loader": "^5.0.0",
+    "i": "^0.3.6",
     "import-inject-loader": "^0.2.4",
     "inject-loader": "^4.0.1",
     "jasmine-core": "^3.2.1",
     "karma": "^3.0.0",
     "karma-chrome-launcher": "^2.2.0",
     "karma-jasmine": "^1.1.2",
     "karma-webpack": "4.0.0-rc.2",
+    "puppeteer": "^1.9.0",
```

Comment:
Using puppeteer for browser (I assumed the build machines wouldn't have chrome installed)